components/automate-chef-io/content/docs/iam-v2-api-reference.md

```diff
@@ -11,7 +11,7 @@ toc = true
 +++
 
 This API reference details Chef Automate IAM v2 features from the command line.
-If you are not already on IAM v2, [upgrade to IAM v2]({{< relref "iam-v2-guide.md#upgrade-to-iam-v2" >}}) first to ensure expected outcomes when using `curl` commands.
+If you are not already on IAM v2, [upgrade to IAM v2]({{< relref "#upgrade-to-iam-v2" >}}) first to ensure expected outcomes when using `curl` commands.
```

Comment:
I fixed the relrefs so that I could build the docs but left the commit separate so that it can be pulled out easily if someone needs the change sooner, or has already fixed it in their branch.